App.js

```diff
@@ -103,8 +102,10 @@ export default function App() {
     return <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>{displaypage}</View>;
   };
 
-  if (!isLoggedIn) {
-    return <LoginPage setLoggedIn={setLoggedIn}/>;
+  if (!isLoggedIn && authPage === 'signup') {
+    return <SignupPage setLoggedIn={setLoggedIn} setAuthPage={setAuthPage} />;
```

Comment:
Not a big issue but I think sign up is two words, so might be worth capitalizing up